### PR TITLE
Remove reintroduced non-null assertions in `Db` calls

### PR DESCRIPTION
**Pull request by ivov**

### CODE REVIEW

File: packages/cli/src/ActiveExecutions.ts [hunk 1]
```diff
@@ -88,7 +86,7 @@ export class ActiveExecutions {
 			};
 
 			// @ts-ignore
```
Comment (BHesseldieck) [inline packages/cli/src/ActiveExecutions.ts:88]: can we remove this?

**Review by BHesseldieck — APPROVED** [review]

File: packages/cli/test/integration/passwordReset.api.test.ts [hunk 2]
```diff
@@ -47,27 +47,34 @@ afterAll(async () => {
...
+		await utils.configureSmtp();
+
+		await Promise.all(
+			[{ email: owner.email }, { email: member.email.toUpperCase() }].map(async (payload) => {
+				const response = await authlessAgent.post('/forgot-password').send(payload);
+
+				expect(response.statusCode).toBe(200);
+				expect(response.body).toEqual({});
+
+				const user = await Db.collections.User!.findOneOrFail({ email: payload.email });
```
Comment (BHesseldieck) [inline packages/cli/test/integration/passwordReset.api.test.ts:70]: shouldn't this be that?
```suggestion
				const user = await Db.collections.User.findOneOrFail({ email: payload.email });
```

File: packages/cli/test/integration/users.api.test.ts [hunk 3]
```diff
@@ -564,9 +521,66 @@ test('POST /users should fail with invalid inputs', async () => {
...
+
+		const response = await authOwnerAgent.post('/users').send([]);
+
+		const { data } = response.body;
+
+		expect(response.statusCode).toBe(200);
+		expect(Array.isArray(data)).toBe(true);
+		expect(data.length).toBe(0);
+
+		const users = await Db.collections.User!.find();
```
Comment (BHesseldieck) [inline packages/cli/test/integration/users.api.test.ts:579]:
```suggestion
		const users = await Db.collections.User.find();
```

**Review by BHesseldieck — APPROVED** [review]

**Review by BHesseldieck — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.175.0`

**Pull request merged**